CHANGELOG.rst

```diff
@@ -1,5 +1,13 @@
 This file includes migration steps for each release of CGXP.
 
+Version 1.6
+===========
+
+1. We start to use the MapFishPrint version 3 then in the ``cgxp_print`` 
```

Comment:
Shouldn't you also add that the form fields (title, comments, legend checkbox) are no longer provided by default and must be described (and translated) in the ``additionalAttributes`` param?